.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,15 @@
+(Make sure your title is either: 'fix', 'chore', or 'feat' then your title. ex: `fix: add new plugin`)
+
+Fixes Issue # (If it doesn't fix an issue then delete this line)
+
+Features Added:
+- Plugin Name (Add links if possible too)
+
+Reasoning:
+List why the feature is needed 
+
+Speed:
```

Comment:
Worth having this as optional? or maybe even....

Show the impact of the speed on NvChad if applicable:

Like a typo fix won't need speed impact.


I do really like having this in the template though, as it keeps a focus on speed.

Reply:
Sure sounds good